src/modules/navigator/rtl_params.c

```diff
@@ -118,9 +118,10 @@ PARAM_DEFINE_FLOAT(RTL_MIN_DIST, 5.0f);
 PARAM_DEFINE_INT32(RTL_TYPE, 0);
 
 /**
- * Half-angle of the RTL cone.
+ * Half-angle of the return to home cone.
  *
- * Defines the half-angle of the cone which defines the vehicle RTL behavior.
+ * Defines the half-angle of a cone centered around the home position that
+ * affects the altitude at which the vehicle returns during return to home.
```

Comment:
Above is OK, but if you choose you can change the last words to "... during the return." to future proof it against rally points

Reply:
@hamishwillee I think I don't quite understand as I'm not at all familiar with rally points. Is the RTL cone only supposed to work for a return to home or also for rally points.

Reply:
@RomanBapst Sorry, please ignore this comment - right now the text is correct. When there are rally points we will revisit. 

Just FYI, a rally point is an alternative return destination that you might end up at instead of home (usually closest rally point or home will be used). I expect that anything it currently says about home will apply to the destination rally point instead - ie cone, but until they are implemented who knows :-)